fastlane/Fastfile

```diff
@@ -55,12 +55,13 @@ platform :android do
   desc "Upload APK to Play Store, in production track with a very small rollout percentage"
   lane :deploy_playstore do
 
-    update_fastlane_release_notes()
-
     props = property_file_read(file: "app/version/version.properties")
     version = props["VERSION"]
+    flversion = convert_version(release_number: version)
     apkPath = "app/build/outputs/apk/play/release/duckduckgo-#{version}-play-release.apk"
 
+    update_fastlane_release_notes(release_number: flversion)
```

Comment:
passing version code in so that `update_fastlane_release_notes` doesn't have to recalculate it based on the value on disk.